certified-connectors/StarRez REST v1/apiDefinition.swagger.json

```diff
@@ -0,0 +1,17799 @@
+                    "type": "string",
+                    "description": "",
+                    "title": "Name First"
+                  },
+                  "NameInitials": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Name Initials"
+                  },
+                  "NameLast": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Name Last"
+                  },
+                  "NameOther": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Name Other"
+                  },
+                  "NamePreferred": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Name Preferred"
+                  },
+                  "NameSharer": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Name Sharer"
+                  },
+                  "NameTitle": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Name Title"
+                  },
+                  "NameWeb": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Name Web"
+                  },
+                  "PinNumber": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "",
+                    "title": "Pin Number"
+                  },
+                  "PortalAuthProviderUserID": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Portal Auth Provider User ID"
+                  },
+                  "PortalEmail": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Portal Email"
+                  },
+                  "Position": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Position"
+                  },
+                  "Previous_EntryStatusEnum": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Previous Entry Status Enum"
+                  },
+                  "TaxExemptionEnum": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Tax Exemption Enum"
+                  },
+                  "Testing": {
+                    "type": "boolean",
+                    "description": "Testing",
+                    "title": "Testing",
+                    "enum": [
+                      true,
+                      false
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        },
+        "operationId": "Select_Entry",
```

Comment:
@Tony747SREZ all operationIds should be in Camel Case. please change to "SelectEntry". Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please fix all operationIds in the swagger file.

Reply:
We will update the operations to remove the underscores.